src/web-components/components/index.astro

```diff
@@ -0,0 +1,5 @@
+---
+import Logo from "./Logo/Logo.astro"
+---
+
+<Logo />
```

Comment:
A fun side effect, no need for `js` files, all the components are defined in `astro` files